lib/runner/eventHub.js

```diff
@@ -0,0 +1,107 @@
+const EventEmitter = require('events');
+
+class NightwatchEventHub extends EventEmitter {
+  emit(eventName, data) {
+    if (this.isAvailable) {
```

Comment:
Since the events published here should not influence the rest of the execution, maybe we should use a try/catch to ignore any errors thrown in the listeners.

Reply:
I attempted to add the try/catch wrapper, but `ESLint` is indicating that it is an unnecessary addition. While it is possible to ignore the `ESLint` suggestion, I would like to hear your opinion on this matter, @beatfactor.